.github/workflows/system-tests.yml

```diff
@@ -7,8 +7,9 @@ on:
       - 1.x
   pull_request:
     paths:
-      - ddtrace/*
-      - setup*
```

Comment:
oh doesn't `setup*` work to match both `setup.py` and `setup.cfg`?

Reply:
Seems like it should according to the last example on https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths .

Reply:
`ddtrace/*` would only match files in `ddtrace/` itself, though, so `ddtrace/**` is needed for anything nested more than one level deep.

Reply:
I changed `setup*` just to be more explicit in case any other file with a `setup` prefix is added in the future